components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -366,6 +366,8 @@ void AdsServiceImpl::MaybeStartBatAdsService() {
 void AdsServiceImpl::StartBatAdsService() {
   CHECK(!IsBatAdsServiceBound());
 
+  VLOG(6) << "Starting Bat Ads Service";
```

Comment:
Do we really need these VLOGs?
https://bravesoftware.slack.com/archives/C7VLGSR55/p1736361619536969

Reply:
Removed